src/shutdown_commands.py

```diff
@@ -1,12 +1,16 @@
+import logging
 import os
 
+logger = logging.getLogger(__name__)
 
 class ShutdownCommands:
     def __init__(self):
         pass
 
     def shutdown(self):
+        logging.debug("shutdown command recieved.")
```

Comment:
@aitorcas23, entiendo la idea, pero creo que el mensaje no es adecuado, lo siento. Esta clase no sabe nada sobre ningún comando/evento, su única función es apagar/reiniciar el dispositivo cuando llamen a sus funciones. El que sabe sobre los comandos es el CommunicationServer y ahí sí que imprimimos un mensaje en el log cuando se recibe un comando ```logger.debug(f"Command '{req_command}' has been requested")```. Si quieres añadir un mensaje aquí, yo optaría por:

```suggestion
        logging.debug("Shutting down the device.")
```